misc/toml-parser/src/main/java/io/ballerina/toml/semantic/ast/TomlKeyNode.java

```diff
@@ -55,6 +55,14 @@ public String name() {
         return String.join(".", list);
     }
 
+    public String originalName() {
```

Comment:
This is asking for trouble, Given now we have two methods `name` and `orignalName`, consumers of the API will get confused what is the correct method to use. This is not a good API design.

Reply:
This is done to make the names a bit consistent with other APIs in ballerina[1]

We can change the method name to unquotedName() to fit the use case. Shall I do that then? 

[1]https://github.com/ballerina-platform/ballerina-lang/blob/c3fdd7f3c41c01c155b8eaac4ad0d5a5f7ceabf1/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/symbols/BSymbol.java#L98

Reply:
Yes, when I see those APIs now it is a bit confusing. Why can't we just return the actual value, instead of the literal value? Most of cases those two are equal. unless we use a quoted identifier.  Do we want to use literal value anywhere?